src/main/modules/Execution.js

```diff
@@ -0,0 +1,41 @@
+var index2 = 0;
+export class Execution {
+    constructor(parserActionsArray, parserActorsArray){
+        this.parserActionsArray = parserActionsArray;
+        this.parserActorsArray = parserActorsArray;
+    }
+
+    execute(parserActionsArray, parserActorsArray){
+        for (var index = 0; index < (parserActionsArray.length); index++) {
+            switch(parserActionsArray[index].actionType){
+            case 'Pick Weapon':
+                for (index2 = 0; index2 < parserActorsArray.length; index2++){
```

Comment:
I think there is a lint error, add an space between the () and {}

Reply:
You are right, not just here, there were a lot of errors like this around all the files, they were already corrected, check them and tell me what do you think.